package.json

```diff
@@ -28,25 +28,24 @@
   "devDependencies": {
     "@babel/core": "^7.18.13",
     "@babel/preset-env": "^7.18.10",
-    "@commitlint/cli": "^17.0.3",
+    "@commitlint/cli": "^17.1.2",
     "@lwc/eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
     "@lwc/jest-utils-lwc-internals": "link:./scripts/jest/utils",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-replace": "^4.0.0",
-    "@rollup/plugin-typescript": "^8.3.4",
-    "@types/acorn": "^4.0.6",
+    "@rollup/plugin-typescript": "^8.4.0",
     "@types/babel__core": "^7.1.19",
     "@types/glob": "^7.2.0",
-    "@types/jest": "^28.1.7",
-    "@types/node": "^18.7.11",
-    "@typescript-eslint/eslint-plugin": "^5.34.0",
-    "@typescript-eslint/parser": "^5.34.0",
+    "@types/jest": "^28.1.8",
+    "@types/node": "^18.7.13",
+    "@typescript-eslint/eslint-plugin": "^5.35.1",
```

Comment:
We also can't upgrade `typescript` because of `@typescript-eslint` complaining:

```
=============                
                                                                                                                                       
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
                                                                                                                                       
You may find that it works just fine, or you may not.                                                                                  
                                                                                                                                       
SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.8.0                                                                                          
                                                                                                                                       
YOUR TYPESCRIPT VERSION: 4.8.2                                                                                                         
                                                                                                                                       
Please only submit bug reports when using the officially supported version.                             
                                                                                                                                       
=============          
```